salt/lax/bot-lax-adaptor.sls

```diff
@@ -186,15 +186,10 @@ uwsgi-bot-lax-adaptor:
         - require:
             - service: uwsgi-bot-lax-adaptor
 
-    # test to ensure service is not serving up 500 responses
-    # the 'listen' statement ensures it runs at the end of a state run
-    # currently doesn't work: due to `listen` it runs 
-    # after the listener_nginx-server-service restart
-    # but it also runs immediately here, and since nginx is not restarted yet, it fails because no one is answering on port 8001 yet
-    #cmd.run:
-    #    - name: |
-    #        curl --silent --include --head --fail {{ apiprotocol }}://{{ apihost }}:8001/ui/
-    #    - listen:
-    #        - service: nginx
-    #        - service: uwsgi-bot-lax-adaptor
+uwsgi-bot-lax-smoke-test:
+    http.wait_for_successful_query:
+        - name: {{ apiprotocol }}://{{ apihost }}:8001/ui/
+        - status: 200
```

Comment:
Could argue that once this open the port and starts answering requests, it should not return anything else than 200?

Reply:
(but it may be too difficult to ensure)